Remove scale calculation in Climacell for cloud cover percentage

apaperclip opened the pull request:
<!--
  You are amazing! Thanks for contributing to our project!
  Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).
-->
## Breaking change
<!--
  If your PR contains a breaking change for existing users, it is important
  to tell them what breaks, how to make it work again and why we did this.
  This piece of text is published with the release notes, so it helps if you
  write it towards our users, not us.
  Note: Remove this section if this PR is NOT a breaking change.
-->


## Proposed change
<!--
  Describe the big picture of your changes here to communicate to the
  maintainers why we should accept this pull request. If it fixes a bug
  or resolves a feature request, be sure to link to that issue in the
  additional information section.
-->
In the climacell updates in 2021.7.0, the new cloud cover entity value was in sub 1 decimal (e.g. ".28" for "28%")
I confirmed by comparing the climacell (tomorrow.io) website cloud cover value with what the entity shows for my area.


## Type of change
<!--
  What type of change does your PR introduce to Home Assistant?
  NOTE: Please, check only 1! box!
  If your PR requires multiple boxes to be checked, you'll most likely need to
  split it into multiple PRs. This makes things easier and faster to code review.
-->

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New integration (thank you!)
- [ ] New feature (which adds functionality to an existing integration)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information
<!--
  Details are important, and help maintainers processing your PR.
  Please be sure to fill out additional details, if applicable.
-->

- This PR fixes or closes issue: fixes #
- This PR is related to issue: 
- Link to documentation pull request: 

## Checklist
<!--
  Put an `x` in the boxes that apply. You can also fill these out after
  creating the PR. If you're unsure about any of them, don't hesitate to ask.
  We're here to help! This is simply a reminder of what we are going to look
  for before merging your code.
-->

- [x] The code change is tested and works locally.
- [ ] Local tests pass. **Your PR cannot be merged unless tests pass**
- [x] There is no commented out code in this PR.
- [ ] I have followed the [development checklist][dev-checklist]
- [ ] The code has been formatted using Black (`black --fast homeassistant tests`)
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

If the code communicates with devices, web services, or third-party tools:

- [ ] The [manifest file][manifest-docs] has all fields filled out correctly.  
      Updated and included derived files by running: `python3 -m script.hassfest`.
- [ ] New or updated dependencies have been added to `requirements_all.txt`.  
      Updated by running `python3 -m script.gen_requirements_all`.
- [ ] Untested files have been added to `.coveragerc`.

The integration reached or maintains the following [Integration Quality Scale][quality-scale]:
<!--
  The Integration Quality Scale scores an integration on the code quality
  and user experience. Each level of the quality scale consists of a list
  of requirements. We highly recommend getting your integration scored!
-->

- [ ] No score or internal
- [ ] 🥈 Silver
- [ ] 🥇 Gold
- [ ] 🏆 Platinum

<!--
  This project is very active and we have a high turnover of pull requests.

  Unfortunately, the number of incoming pull requests is higher than what our
  reviewers can review and merge so there is a long backlog of pull requests
  waiting for review. You can help here!
  
  By reviewing another pull request, you will help raise the code quality of
  that pull request and the final review will be faster. This way the general
  pace of pull request reviews will go up and your wait time will go down.
  
  When picking a pull request to review, try to choose one that hasn't yet
  been reviewed.

  Thanks for helping out!
-->

To help with the load of incoming pull requests:

- [ ] I have reviewed two other [open pull requests][prs] in this repository.

[prs]: https://github.com/home-assistant/core/pulls?q=is%3Aopen+is%3Apr+-author%3A%40me+-draft%3Atrue+-label%3Awaiting-for-upstream+sort%3Acreated-desc+review%3Anone

<!--
  Thank you for contributing <3

  Below, some useful links you could explore:
-->
[dev-checklist]: https://developers.home-assistant.io/docs/en/development_checklist.html
[manifest-docs]: https://developers.home-assistant.io/docs/en/creating_integration_manifest.html
[quality-scale]: https://developers.home-assistant.io/docs/en/next/integration_quality_scale_index.html
[docs-repository]: https://github.com/home-assistant/home-assistant.io

apaperclip committed:
Remove scale calculation for cloud cover percentage

raman325 reviewed (commented):
Thanks, guess I got mixed up. You will have to update the test for this sensor. In addition, can you remove these lines and update the corresponding weather entity test as well? https://github.com/home-assistant/core/blob/dev/homeassistant/components/climacell/weather.py#L210-L211

apaperclip commented:
Welcome!  Sorry for missing those, I was a little too eager.  I'll take a look!

apaperclip commented:
I think I need to close this out and start over with a branch for the changes. After your review I started to build a proper dev environment with  a devcontainer and use tests, etc.   On commit from the dev environment  it seems to not like that I'm going right to dev. I knew better and should have done that anyway.

- Based on your feedback, I removed the two lines you linked to, 
- Updated this to 100 and all 18 tests pass https://github.com/home-assistant/core/blob/4e848f60c4b17859f12f2d7ff6c2d8a47d15e292/tests/components/climacell/test_sensor.py#L185

- I also updated this  to 100 (in both places) and those two tests pass. https://github.com/home-assistant/core/blob/4e848f60c4b17859f12f2d7ff6c2d8a47d15e292/tests/components/climacell/test_weather.py#L394 

I _think_ that's it.  I'm new to this so thank you for reviewing. I realize its probably faster for you to just update, but I'm happy to put some time into it and appreciate your time.  If there is any thing else I'll make those updates as well and can submit all of it in a new PR.

raman325 commented:
that's great! So typically you would want to keep your `dev` branch clean so you can pull the latest changes from upstream (home-assistant/core). But that doesn't make your code unmergeable as is, so you can leave it as is if you'd like and we can still merge from this branch. You'll just have to fix your dev branch after, which you would have to do anyway at this point

raman325 commented:
oh I see, now that you've set up the environment, pre-commit may be preventing you from pushing more to `dev`. I'm sure there's a way to get past that but you may be better off just creating a new branch from here and then committing those changes.

While in your local repo folder:
`git checkout -b my_new_branch`
`git add .`
`git commit -m "Fix tests"`
`git push origin`

The last command will fail but it will tell you the command to push your new branch up to github
